### ISSUE TITLE
I keep getting a 'PackageNotFoundError' error.

### ISSUE THREAD

**Issue by LloydValentine** [gingeleski/keyword-captain#2]
I open up powershell, I use the cd command to get to the folder that keyword captain master is located, which is on my desktop.  Once I get to the "conda create..." step, all I receive is a 'PackageNotFoundError...', message.  I've tried running Powershell as adminastrator, I tried to "Unblock" each file individually, as I figure windows was blocking the files since it keeps saying "these files came from another computer" or something like that.

I am posting a screenshot so you can see what it going on.  Any help is appreciated, thanks.

![screenshot 2017-03-27 22 33 53](https://cloud.githubusercontent.com/assets/26758350/24438909/ba2d8a82-1418-11e7-80d8-3342b5421d99.png)

**Comment by gingeleski** [comment]
I haven't been able to reproduce your issue but am getting a different issue stemming from package-list.txt

While I work through that - because I think it may solve yours as well - what happens when you run `.\run_tests.ps1` ?

(cd into the folder and execute that instead of the `conda ...` line)

**Comment by LloydValentine** [comment]
Hmm, I see, thanks for checking it out for me.  I ran the command and I received this error:

.\run_tests.ps1 : File C:\Users\LamontLloyd\Desktop\keyword-captain-master\run_tests.ps1 cannot be loaded because
running scripts is disabled on this system. For more information, see about_Execution_Policies at
http://go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ .\run_tests.ps1
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess

**Comment by LloydValentine** [comment]
I'm reading about execution policies and they are all "Unsigned" except for my LocalUser which is restricted.  I can change my LocalUser to "Bypass" but it still won't open the package list.

UPDATE:  I set all Policies to Unrestricted except for MachinePolicy and UserPolicy which tells me I have to do that via Group Policy, but I have no clue how to do this.

**Comment by gingeleski** [comment]
Running Powershell as admin, try `set-executionpolicy -executionpolicy unrestricted`

**Comment by LloydValentine** [comment]
Ok I tried the command and I still have undefined for Machine Policy , User Policy, and Process.  I have Unrestricted for Local Policy and Current User Policy.  When I try to create the new environment I still see "PackageNotFoundError".  I have a screenshot below.  

 I probably just need to learn more about powershell in general.  Thanks for all the help.

**Comment by LloydValentine** [comment]
Is there a way to turn this into an excecutionable (.exe) file?

**Comment by gingeleski** [comment]
Yes, [py2exe](http://www.py2exe.org/) can allegedly create _executables_, but I anticipate a lot of trial and error in this case. The bot has advanced dependencies and, as I understand it, what py2exe does is convert Python (interpreted) to C (compiled) to executable machine code. That will doubtfully go smoothly, probably be a time sink.

I've had success on [another project](https://github.com/gingeleski/odds-portal-scraper/releases/tag/v1.0) in zipping up dependencies with the software, calling that a release. In this case, dependency management must first be changed from conda to pip + virtualenv. I'm more familiar with those, just avoided them on this because conda already had computer vision libraries.

**TL;DR** - I'm working on something comparable. It's not the most glamorous solution but this project is not my top priority.

**Comment by gingeleski** [comment]
I'm not having luck getting one dependency (pillow) running in a virtualenv. But, since you have Anaconda Python 3 installed already, here's a quick and dirty way things should work.

Running Powershell as administrator...

1. cd into the repo folder, following the instructions [here](https://github.com/gingeleski/keyword-captain/wiki/Setup-for-Laypeople#3-enter-the-source-code-folder-using-powershell) again if needed.
2. ```pip install pyautogui```
3. ```cd tests```
4. Now we'll run some tests, these can prove this is working for you now. Expected results should match actual results.
5. ```python .\test_integration_1.py```
6. ```python .\test_integration_2.py```
7. ```python .\test_integration_3.py```
8. Good? Good. Now we can run the actual bot.
9. You'll need a Scrabble Boggle board open in your browser first, open a practice game maybe.
10. ```cd ..\src```
11. ```python .\runScrabbleBoggle.py```
12. When you want to stop the bot, put your mouse into the upper leftmost corner of the screen. Or doing Ctrl+C might work.

Open a separate issue (and a new screenshot if applicable) for any trouble. ✌️
